src/Sdk/DTLogging/Logging/ValueEncoders.cs

```diff
@@ -60,6 +60,20 @@ public static String XmlDataEscape(String value)
             return SecurityElement.Escape(value);
         }
 
+        public static String TrimDoubelQuoteDataEscape(String value)
```

Comment:
How about `TrimQuotes` or `TrimDoubleQuotes`?

"Data" is terminology specific to URI, to distinguish between a segment within the URI vs the whole thing.

Reply:
And "Escape" terminology doesnt apply here either

Reply:
👍